frontend/pages/DashboardPage/cards/ChartCard/ChartCard.tsx

```
@@ -45,12 +45,11 @@ const DATASETS: IDataSet[] = [
     defaultChartType: "checkerboard",
     description: (
       <>
-        The number of hosts detected online during a given hour.
+        The number of hosts detected online (checking in to Fleet) during 
```

ESLint

[error] Insert `a`

(prettier/prettier)